conformance/fhir-ig-davinci-pdex-plan-net/src/main/java/com/ibm/fhir/ig/davinci/pdex/plan/net/util/IndexGenerator.java

```diff
@@ -23,7 +23,6 @@
 public class IndexGenerator {
     public static void main(String[] args) throws Exception {
         buildAndWriteIndex("hl7/fhir/us/davinci-pdex-plan-net");
-        buildAndWriteIndex("hl7/fhir/uv/vhdir");
```

Comment:
not related to the other changes, but thought I might as well include it anyway

Reply:
sounds good